config.lua.dist

```diff
@@ -80,6 +80,48 @@ freeDepotLimit = 2000
 premiumDepotLimit = 10000
 depotBoxes = 20
 
+-- Augments System (Get more info in: https://github.com/opentibiabr/canary/pull/2602)
+--
+-- the key value of the key "augments" is to enable (1) and disable (0) the augments of the item.
+-- The augments should be structered in this way on items.xml:
+--
+-- <attribute key="augments" value="1">
+--   <attribute key="spell name" value="augment type" />
+-- </attribute>
+--
+-- Example of augment that have default value:
+--
+-- <attribute key="augments" value="1">
+--   <attribute key="energy wave" value="strong impact" />
+-- </attribute>
+--
+-- Example of augment that don't have default value, the value must be set:
+--
+-- <attribute key="augments" value="1">
+--   <attribute key="energy wave" value="life leech">
+--     <attribute key="value" value="5" />
+--   </attribute>
+-- </attribute>
+--
+-- NOTE: augments type available:
+-- * "critical extra damage",
+-- * "life leech", 
+-- * "mana leech",
+-- * "cooldown",
+-- * "increased damage", (don't need to set value)
+-- * "strong impact", (don't need to set value)
+-- * "powerful impact" (don't need to set value)
```

Comment:
Things that are not related to the config are better to extract in a separate "readme.md" or document on the wiki, as this will "dirty" the config.lua a lot for things that have no direct connection with the configuration file

Reply:
Elson already warned me, I'll be removing and adding to the PR.